vendor/github.com/containerd/cgroups/blkio.go

```diff
@@ -0,0 +1,340 @@
+// +build !windows
```

Comment:
Note I had to add the `// +build !windows` to these due to the go mod import issues with `metrics.proto`. But I am going to move the `metrics.proto` in a separate pr in `github.com/containerd/cgroups` so that we dont need to import any code to import the types.

Reply:
Once: https://github.com/containerd/cgroups/pull/96 merges I will re-vendor this with the correct package for just the proto file.